### PR TITLE
feat: solana send for coin support and cli

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6587 Add Solana send on coin support and CLI](https://app.clickup.com/t/9002019994/PRF-6587)

File: packages/coin-support-solana/src/services/api/transaction.ts [hunk 1]
```diff
@@ -32,3 +32,43 @@ export const getTransactions = async (params: {
...
+
+export const getFees = async (params: { assetId: string }) => {
+  const url = `${baseURL}/fees`;
+
+  const query: Record<string, any> = {
+    ...params,
+    responseType: 'v2',
+    network: solanaCoinList[params.assetId].network,
+  };
+  delete query.assetId;
```
Comment (irshadCypherock) [inline packages/coin-support-solana/src/services/api/transaction.ts]:
```suggestion
  const query: Record<string, any> = {
    responseType: 'v2',
    network: solanaCoinList[params.assetId].network,
  };
```

File: packages/coin-support-solana/src/index.ts [hunk 2]
```diff
@@ -40,24 +51,32 @@ export class SolanaSupport implements CoinSupport {
...
 
-  public broadcastTransaction(): Promise<ITransaction> {
-    throw new Error(`Method not implemented`);
+  public broadcastTransaction(
+    params: IBroadcastSolanaTransactionParams,
+  ): Promise<ITransaction> {
+    return operations.broadcastTransaction(params);
   }
 
   public validateAddress(): boolean {
```
Comment (irshadCypherock) [inline packages/coin-support-solana/src/index.ts]: Implement validateAddress

**Review by irshadCypherock — CHANGES_REQUESTED** [review]
- Txn Fees is not displayed on cli when making a txn

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**